src/prefect/settings.py

```diff
@@ -835,6 +835,12 @@ def default_cloud_ui_url(settings, value):
 )
 """The API's port address (defaults to `4200`)."""
 
+PREFECT_ORION_API_KEEPALIVE = Setting(
+    int,
+    default=5,
+)
+"""The API's keep alive timeout (defaults to `5`)."""
```

Comment:
It may be good to explain what this is for and when it is useful. Perhaps link to the Uvicorn documentation.

Reply:
Will do. I was not sure if it was okay to add an external link. I added it and then removed it before submitting.

Reply:
I have added additional text. Hopefully, that is more helpful.